src/PageContent/Calendar/EventCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import './EventCard.css';
+import green_truck from '../../icons/green_truck.svg';
+import grey_truck from '../../icons/grey_truck.svg';
+
+class EventCard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
- these info would probably be passed down as `props` rather than `state`?
- please create enums for `eventType`, add to `Enums.js`
```
exports.DeliveryType = {
    RECURRING = 'recurring',
    EMERGENCY = 'emergency'
};
```
- `eventStatus` could be changed to `isPast` that's `true`/`false`
- for `eventTime`, please take in `startTime` and `endTime` instead